app/bundles/ReportBundle/Form/Type/DynamicFiltersType.php

```diff
@@ -58,6 +58,8 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                         $args['html5']  = false;
                         $args['format'] = 'y-MM-dd';
                         $args['attr']['class'] .= ' datepicker';
+                        if (!isset($options['data'][$definition['alias']]) && (empty($filter['value']) || '' === $filter['value'])) {
+                        }
```

Comment:
```suggestion

```
The conditional block is empty. It should be removed if unnecessary or completed with logic.